plugin.py

```diff
@@ -77,3 +79,70 @@ def _cleanup_cache(cls) -> None:
         """ Clean up this plugin's cache directory. """
 
         shutil.rmtree(get_plugin_storage_dir(), ignore_errors=True)
+
+
+class LspTexlabForwardSearchCommand(LspTextCommand):
+
+    def run(self, edit):
+        session = self.session_by_name(PLUGIN_NAME)
+        if not session:
+            return
+        row, col = self.view.rowcol(next(iter(self.view.sel())).a)
+        params = {
+            "textDocument": {
+                "uri": uri_from_view(self.view)
+            },
+            "position": {
+                "line": row,
+                "character": col
+            }
+        }
```

Comment:
There is `text_document_position_params` in LSP.plugin.core.views which handles this.

```
params = text_document_position_params(self.view, self.view.sel().a)
```